lib/sharing/editor_ui.dart

```diff
@@ -211,18 +216,16 @@ abstract class EditorUi {
         clearOutput();
 
         await executionService.execute(
-          context.htmlSource,
-          context.cssSource,
-          response.result,
-          modulesBaseUrl: response.modulesBaseUrl,
-          addRequireJs: true,
-          addFirebaseJs: shouldAddFirebaseJs,
-          // TODO(srawlins): Determine if we need to destroy the frame when
-          // changing channels.
-          // TODO(ryjohn) Determine how to preserve the iframe
-          // https://github.com/dart-lang/dart-pad/issues/2269
-          destroyFrame: true,
-        );
+            context.htmlSource, context.cssSource, response.result,
+            modulesBaseUrl: response.modulesBaseUrl,
+            addRequireJs: true,
+            addFirebaseJs: shouldAddFirebaseJs,
+            // TODO(srawlins): Determine if we need to destroy the frame when
+            // changing channels.
+            // TODO(ryjohn) Determine how to preserve the iframe
+            // https://github.com/dart-lang/dart-pad/issues/2269
+            destroyFrame: true,
+            canvasKitBaseUrl: _createCanvasKitBaseUrl(version.engineVersion));
```

Comment:
Won't this break stable since its engine versions don't have `canvaskit.js` stored under their hash yet?

Reply:
Could we add a `legacy` flag here – and just load it from the "old" URL?

Reply:
Good catch, fixed in [b1f3d1a](https://github.com/dart-lang/dart-pad/pull/2490/commits/b1f3d1ae3b3c8669ad48c0107704847f8fc2b8d0)